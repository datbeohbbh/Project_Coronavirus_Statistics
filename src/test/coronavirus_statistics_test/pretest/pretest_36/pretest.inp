50 100
q 1 3 191
? 7
q 1 13 342
? 6
q 1 25 368
q 1 26 358
q 1 33 332
q 3 13 38
? 4 1
q 1 36 78
q 1 41 133
? 4 3
? 4 2
? 7
q 1 40 360
q 1 43 207
q 1 19 101
q 1 9 198
? 6
? 7
q 1 49 295
q 2 33 139
? 7
q 1 5 388
q 1 45 86
q 1 47 369
q 3 47 263
q 1 32 396
? 7
? 6
q 1 39 133
q 2 36 46
? 7
? 4 16
? 6
q 1 18 61
q 2 13 135
? 7
q 1 50 446
q 2 18 52
? 4 2
? 7
? 6
q 1 14 359
q 3 47 101
? 4 1
? 7
? 4 13
q 2 9 45
q 1 16 394
? 4 16
q 1 15 251
? 4 7
q 3 41 69
? 4 22
q 2 18 3
q 3 26 56
? 7
q 3 50 38
? 4 9
? 6
q 2 47 1
? 4 4
q 1 1 493
q 3 18 6
q 1 31 111
q 2 43 33
? 7
q 2 13 100
? 6
? 4 16
q 1 22 400
q 1 16 454
q 1 29 401
q 1 8 409
q 3 1 376
q 2 50 389
? 4 19
q 1 5 263
q 2 25 342
q 3 39 24
? 4 11
? 6
? 4 3
? 4 22
? 6
? 4 11
? 4 10
? 7
? 4 14
? 6
? 6
? 6
? 4 26
? 4 21
? 4 1
? 6
? 4 4
? 6
? 6