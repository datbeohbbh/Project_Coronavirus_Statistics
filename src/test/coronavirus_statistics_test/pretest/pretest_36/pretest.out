Query ? 7 :
Currently, no deaths have been recorded

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 1 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 191
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 191
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 41 
- Information of the city 41 -
-> Number of current patients : 133
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 13
Quantity: 38 people

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 7 :
The city with the highest death is city with the index 13
Quantity: 38 people

Query ? 7 :
The city with the highest death is city with the index 13
Quantity: 38 people

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 263 people

Query ? 6 :
The city with the highest recovery is city with the index 33
Quantity: 139 people

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 263 people

Query ? 4 16 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 388
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 33
Quantity: 139 people

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 263 people

Query ? 4 2 :
The answer is the city with the index 36 
- Information of the city 36 -
-> Number of current patients : 32
-> Has recovered : 46
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 263 people

Query ? 6 :
The city with the highest recovery is city with the index 33
Quantity: 139 people

Query ? 4 1 :
The answer is the city with the index 47 
- Information of the city 47 -
-> Number of current patients : 5
-> Has recovered : 0
-> Was dead : 364

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 364 people

Query ? 4 13 :
The answer is the city with the index 49 
- Information of the city 49 -
-> Number of current patients : 295
-> Has recovered : 0
-> Was dead : 0

Query ? 4 16 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 360
-> Has recovered : 0
-> Was dead : 0

Query ? 4 7 :
The answer is the city with the index 41 
- Information of the city 41 -
-> Number of current patients : 133
-> Has recovered : 0
-> Was dead : 0

Query ? 4 22 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 446
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 364 people

Query ? 4 9 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 169
-> Has recovered : 135
-> Was dead : 38

Query ? 6 :
The city with the highest recovery is city with the index 33
Quantity: 139 people

Query ? 4 4 :
The answer is the city with the index 41 
- Information of the city 41 -
-> Number of current patients : 64
-> Has recovered : 0
-> Was dead : 69

Query ? 7 :
The city with the highest death is city with the index 47
Quantity: 364 people

Query ? 6 :
The city with the highest recovery is city with the index 13
Quantity: 235 people

Query ? 4 16 :
The answer is the city with the index 14 
- Information of the city 14 -
-> Number of current patients : 359
-> Has recovered : 0
-> Was dead : 0

Query ? 4 19 :
The answer is the city with the index 40 
- Information of the city 40 -
-> Number of current patients : 360
-> Has recovered : 0
-> Was dead : 0

Query ? 4 11 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 117
-> Has recovered : 0
-> Was dead : 376

Query ? 6 :
The city with the highest recovery is city with the index 50
Quantity: 389 people

Query ? 4 3 :
The answer is the city with the index 25 
- Information of the city 25 -
-> Number of current patients : 26
-> Has recovered : 342
-> Was dead : 0

Query ? 4 22 :
The answer is the city with the index 22 
- Information of the city 22 -
-> Number of current patients : 400
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 50
Quantity: 389 people

Query ? 4 11 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 117
-> Has recovered : 0
-> Was dead : 376

Query ? 4 10 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 111
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 1
Quantity: 376 people

Query ? 4 14 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 191
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 50
Quantity: 389 people

Query ? 6 :
The city with the highest recovery is city with the index 50
Quantity: 389 people

Query ? 6 :
The city with the highest recovery is city with the index 50
Quantity: 389 people

Query ? 4 26 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 848
-> Has recovered : 0
-> Was dead : 0

Query ? 4 21 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 396
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 47 
- Information of the city 47 -
-> Number of current patients : 4
-> Has recovered : 1
-> Was dead : 364

Query ? 6 :
The city with the highest recovery is city with the index 50
Quantity: 389 people

Query ? 4 4 :
The answer is the city with the index 36 
- Information of the city 36 -
-> Number of current patients : 32
-> Has recovered : 46
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 50
Quantity: 389 people

Query ? 6 :
The city with the highest recovery is city with the index 50
Quantity: 389 people

